lib/Backend/GlobOptFields.cpp

```diff
@@ -1170,6 +1170,9 @@ GlobOpt::ProcessPropOpInTypeCheckSeq(IR::Instr* instr, IR::PropertySymOpnd *opnd
                 opnd->SetSlotIndex(slotIndex);
                 opnd->SetUsesAuxSlot(auxSlot);
 
+                opnd->GetObjTypeSpecInfo()->SetSlotIndex(slotIndex);
```

Comment:
Does it make sense to have the methods on Opnd themselves set the respective fields on its objTypeSpecFldInfo?

Reply:
I see what you're saying, but the normal flow is from ObjTypeSpecFldInfo, which is set up at work item creation time, to Opnd, which is JIT time.

Reply:
(And I say that recognizing that the way it's written there isn't as clean a division between the two as we might like.)

Reply:
Yeah, I see what you mean. Approved as is.